pages/account.vue

```diff
@@ -211,23 +198,13 @@ const { handleError } = useErrorHandler()
 useHead({
   title: $t('account_page_title'),
 })
-
-const isCrispLoaded = ref(false)
-// XXX: `id` in `useScriptCrisp()` is broken, it always returns `'crisp'` even `id` is provided and it is also not reactive 💩
-const crispId = computed(() => config.public.scripts.crisp.id)
+const crispId = String(config.public.scripts.crisp.id)
 const crispChatURL = computed(() => {
   const url = new URL('https://go.crisp.chat/chat/embed')
-  url.searchParams.set('website_id', crispId.value || '5c009125-5863-4059-ba65-43f177ca33f7')
+  url.searchParams.set('website_id', (crispId || '5c009125-5863-4059-ba65-43f177ca33f7'))
```

Comment:
Unnecessary brackets 
```suggestion
  url.searchParams.set('website_id', crispId || '5c009125-5863-4059-ba65-43f177ca33f7')
```